BASE/src/Microsoft.ApplicationInsights/Channel/Transmission.cs

```diff
@@ -209,13 +214,23 @@ public virtual async Task<HttpWebResponseWrapper> SendAsync()
                         }
                     }
                     catch (OperationCanceledException)
-                    {                        
+                    {
                         wrapper = new HttpWebResponseWrapper
                         {
                             StatusCode = (int)HttpStatusCode.RequestTimeout,
                         };
                     }
 
+                    try
+                    {
+                        // Initiates event notification to subscriber with Transmission and TransmissionStatusEventArgs.
+                        this.TransmissionStatusEvent?.Invoke(this, new TransmissionStatusEventArgs(wrapper));
+                    }
+                    catch
+                    {
+                        // Swallow any exception here as this code is for tracing purposes only and should never throw.
```

Comment:
we should log here and mark the event UserActionable. Because if this fails, its mostly user error and they should know it. SIlently failing is risky here.